tests/engines/registries/acceptance/draft/draft-test.ts

```diff
@@ -503,6 +506,6 @@ module('Registries | Acceptance | draft form', hooks => {
         await fillIn('input[name="title"]', 'A non-empty, but still vague title');
         await click('[data-test-goto-next-page]');
         assert.dom('[data-test-link="metadata"] > [data-test-icon]')
-            .hasClass('fa-check-circle-o', 'metadata page is now valid');
+            .hasClass('fa-exclamation-circle', 'metadata page is now invalid');
```

Comment:
Here, you're changing the nature of the test. This was checking if validation status gets updated from invalid to valid because user fixed the issue with metadata title. I understand you're adding validation for subjects with this PR which affects this test, but we have to keep tests meaningful.

Reply:
I think @futa-ikeda is working on updating the tests. He can update this to a more meaningful test for the draft page.

Reply:
Yea, I will be updating the tests to look for validation messages for subjects, Node license fields, etc. If you guys think of any testing scenarios, please jot them down on the notion ticket pls 🙌